src/simularium/VisGeometry.ts

```diff
@@ -477,6 +478,26 @@ class VisGeometry {
         this.camera.position.copy(newPosition);
     }
 
+    public setPanningMode(pan: boolean): void {
+        if (!pan) {
+            this.controls.enablePan = false;
+            this.controls.enableRotate = true;
+            this.controls.mouseButtons = {
+                LEFT: MOUSE.ROTATE,
+                MIDDLE: MOUSE.DOLLY,
+                RIGHT: MOUSE.PAN,
```

Comment:
Whoa, we can do left = rotate and right = pan? This almost sounds like we can just have this and not bother with hot keys/buttons :P